jvm-packages/xgboost4j-example/src/main/scala/ml/dmlc/xgboost4j/scala/example/CrossValidation.scala

```diff
@@ -40,7 +40,6 @@ object CrossValidation {
     // set additional eval_metrics
     val metrics: Array[String] = null
 
-    val evalHist: Array[String] =
-      XGBoost.crossValidation(trainMat, params.toMap, round, nfold, metrics)
+    XGBoost.crossValidation(trainMat, params.toMap, round, nfold, metrics)
```

Comment:
Since this is an example, maybe it's useful to print it?

Reply:
I would not to refine the tests in this PR. Maybe have the follow up after merging my all small PRs for reworking jvm